src/theme.ts

```diff
@@ -243,6 +243,30 @@ export async function setTheme(theme?: string): Promise<void> {
         setCustomThemeVars(customTheme);
     }
 
+    // Clear the previous theme used
+    document.body.className = "";
+    // If we're not using the system default we want to add a class to the
+    // top-most element, to override the CSS properties
+    if (!SettingsStore.getValue("use_system_theme")) {
+        if (theme.indexOf("dark") > -1) {
+            if (theme.indexOf("high-contrast") > -1) {
+                document.body.classList.add("cpd-theme-dark-hc");
```

Comment:
Originates from https://github.com/vector-im/compound/blob/main/design-tokens/assets/web/css/cpd-dark-hc.css . Will probably want to expose those values in the JS package to avoid having hardcoded values.

Reply:
Do you want to do that change before getting this PR merged, or do you want to do it later on?